vm/src/stdlib/pwd.rs

```diff
@@ -2,6 +2,7 @@
 
 pub(crate) use pwd::make_module;
 
+#[allow(unused)]
```

Comment:
Because the conditions in `stdlib/mod.rs` must be aligned, this `allow` will not be used.

Reply:
it's only being used on android.
I can do what coderabbitai is suggesting if that's ok